src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,7 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
-        "symfony/expression-language": "",
+        "symfony/expression-language": "For using the DependencyInjection YAML/XML file loaders and PHP/YAML/XML dumpers",
```

Comment:
@jvasseur You're right, thank you.
Changed to "For using expression matching in service container configuration files"